nvflare/job_config/fed_job.py

```diff
@@ -218,6 +276,32 @@ def to(
         if self._components:
             self._add_referenced_components(obj, target)
 
+    def to_server(
+        self,
+        obj: Any,
+        tasks: List[str] = None,
+        gpu: Union[int, List[int]] = None,
+        filter_type: FilterType = None,
+        id=None,
+    ):
+        if isinstance(obj, Executor):
+            raise ValueError("Use `job.to(executor, <client_name>)` or `job.to_clients(executor)` for Executors.")
+
+        self.to(obj=obj, target=SERVER_SITE_NAME, tasks=tasks, gpu=gpu, filter_type=filter_type, id=id)
+
+    def to_clients(
+        self,
+        obj: Any,
+        tasks: List[str] = None,
+        gpu: Union[int, List[int]] = None,
```

Comment:
The function looks right. However, the "to_server()" and "to_clients" are only used to support the deploy to @ALL sites use case. However, this is not obvious to the users. I think it's better to combine these two methods to "to_all_sites(target="Server/Client")". The "target" parameter indicates where the component deploys to. And the name "to_all_sites" clearly shows this method intents to do. Otherwise, it's easy to confuse with the "to()" method.

Reply:
Discussed with @holgerroth on this. The names and meanings are subjective. We will keep the implementation for now. And use the documentation to explain to the users.